### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by FireCulex** [Telecommunication-Telemedia-Assessment/bitstream_mode3_p1204_3#25]
![wguoCn9A3q](https://user-images.githubusercontent.com/9983630/131282245-299f0897-1d21-4b0a-9246-23b6bc17eec7.png)

Between analyzing frames and stats output it appears only a single core is being utilized the entire time. 

```
real	5m33.407s
user	5m24.300s
sys	0m8.615s
```
just to analyze a 10 second video.

**Comment by stg7** [comment]
Yes, sorry for this, the reference implementation is not optimized for speed, and the bitstream parser may create quite long reports (that are even compressed and include far more features that are required for the model itself).

To speed up the calculation it would require developing a reduced video parser, we may check this in the future.

Best
Steve

**Comment by stg7** [comment]
also checkout LDJSON report generation
